scss/detail.scss

```diff
@@ -1,4 +1,16 @@
+.container {
+  align-items: center;
+  background: $white-smoke;
+  display: flex;
+  flex-direction: row;
```

Comment:
Default flex direction is row, therefore, it is not necessary to add it.

Reply:
Thanks, it was something I didn't know, I'll try to keep it in mind